1-Draft/RFC00XX-Update-Error-View.md

```diff
@@ -0,0 +1,229 @@
+---
+RFC: Update $ErrorView with simplified views for Error Messages
+Author: Jason Helmick
+Status: Draft
+Version: 0.0.0
+Area: PowerShell
+Comments Due: 10/31/2019
+---
+
+# Update $ErrorView with simplified views for Error Messages
+
+When an error occurs in PowerShell, the customers on-screen error message experience currently
+provides a level of detail that obscures the exception message from being recognized and read by
+new or occasional PowerShell users. The addition of a simplified error view will improve both
+comprehension and troubleshooting experience. A new cmdlet 'Resolve-ErrorRecord' will provide
```

Comment:
I am updating the RFC to reflect a change in the cmdlet name.  After reviewing all of the community feedback and examining the current ecosystem of cmdlets, we are changing Resolve-ErrorRecord to Get-Error.